packages/engine/src/character/CharacterInputSchema.ts

```diff
@@ -202,7 +201,7 @@ const changeCameraDistanceByDelta: InputBehaviorType = (
   const inputPrevValue = (inputComponent.prevData.get(inputKey)?.value as number) ?? 0
   const value = inputValue.value as number
 
-  const scrollDelta = Math.min(1, Math.max(-1, value - inputPrevValue)) * (isMobile ? 0.25 : 1)
+  const scrollDelta = Math.min(1, Math.max(-1, value - inputPrevValue)) * (window?.innerWidth <= 768 ? 0.25 : 1)
```

Comment:
Scroll delta has nothing to do with window size. The feature we should be detecting here is likely related to smooth scrolling capability, or input source (e.g., scroll vs pinch gesture).